File: easy/1984/main.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        diff = float("inf")
        nums.sort()
        for i in range(len(nums)-k+1) :
            diff = min (diff, nums[i+k-1]-nums[i])

        return diff
        

File: easy/1984/main.cpp
class Solution {
public:
    int minimumDifference(vector<int>& nums, int k) {
      int diff = INT_MAX;
      sort(nums.begin(), nums.end());
      for (int i =0 ; i+k-1 < nums.size(); i++) {
        diff = min(diff, nums[i+k-1]-nums[i]);
      }
        
      return diff;
    }
};
